src/components/HeroBanner.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
```

Comment:
No need to explicitly import react

Reply:
I always use ES7 to generate these components that's why 😁😁. Will probably start deleting it. No worries